package.json

```diff
@@ -20,18 +20,21 @@
   },
   "homepage": "https://github.com/micnews/miclint#readme",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node bin.js bin.js index.js options.js"
```

Comment:
@kesla sorry for delayed response, can it not support globs? `node bin.js *.js`?